lib/generate-packets.js

```diff
@@ -37,4 +37,10 @@ async function triggerTabComplete (server, bot) {
   await p
 }
 
+async function triggerChat (server, bot) {
+  const p = once(bot, 'message')
```

Comment:
isnt there a chat event too? maybe we can promise.all with that one too

[ref](https://minecraft-data.prismarine.js.org/?v=1.16.1&d=protocol#toClient_chat)

Reply:
one is enough no ?

Reply:
ah nvm, maybe i see now that mineflayer's message event is triggered by the nmp chat event